tools/interface/ontology.py

```diff
@@ -46,16 +49,50 @@ def GetEntitiesFromFields(self, field_list: List[StandardField], general_type: s
         """
         pass
 
-    def IsTypeValid(self, entity_type: EntityType) -> bool:
-        """Validates the existence of a type in the ontology"""
-        pass
-
     def IsFieldValid(self, field: StandardField) -> bool:
         """A method to validate a field name against the ontology"""
         pass
 
-    def IsSubfieldValid(self, subfield: StandardSubfield) -> bool:
-        """A method to validate a subfield name against the ontology"""
-        pass
 
+class StandardField(object):
+    """
+    A class to represent a generic field without increment or optionality
+
+    Args:
+        namespace_name: the name of the field's namespace as a string
+        field_name: the name of the field as a string
+
+    Attributes:
+        namespace: the name of the namespace as a string
+        name: the field name as a string
+
+    returns:
+        An instance of the StandardField class
 
+    """
+
+    def __init__(self, namespace_name: str, field_name: str):
```

Comment:
sticking with the convention in https://docs.google.com/spreadsheets/d/1ePTg7HVv-eUU1Lc60dW3tpmaPwJMUy0ZvJRmNaOsMME/edit#gid=0  I would call this "standard_field__name"